apps/test/unit/codebridge/FileBrowser/prompts/openNewFolderPromptTest.ts

```diff
@@ -13,126 +13,6 @@ import {EVENTS} from '@cdo/apps/metrics/AnalyticsConstants';
 
-/*
-type OpenNewFilePromptArgsType = {
-  parentId: FolderId;
-  appName: string;
-  dialogControl: DialogControlInterface;
-  newFolder: NewFolderFunction;
-  projectFolders: ProjectType['folders'];
-};
-*/
-
-/*
-export interface MultiFileSource {
-  folders: Record<FolderId, ProjectFolder>;
-  files: Record<FileId, ProjectFile>;
-  openFiles?: FileId[];
-}
-
-export interface ProjectFile {
-  id: FileId;
-  name: string;
-  language: string;
-  contents: string;
-  open?: boolean;
-  active?: boolean;
-  folderId: string;
-  type?: ProjectFileType;
-}
-
-export enum ProjectFileType {
-  STARTER = 'starter',
-  SUPPORT = 'support',
-  VALIDATION = 'validation',
-}
-
-export interface ProjectFolder {
-  id: FolderId;
-  name: string;
-  parentId: string;
-  open?: boolean;
-}*/
-
-/*
-export const openNewFolderPrompt = async ({
-  parentId = DEFAULT_FOLDER_ID,
-  appName,
-  dialogControl,
-  newFolder,
-  projectFolders,
-}: OpenNewFilePromptArgsType) => {
-  const results = await dialogControl.showDialog({
-    type: DialogType.GenericPrompt,
-    title: codebridgeI18n.newFolderPrompt(),
-    validateInput: (folderName: string) => {
-      if (!folderName.length) {
-        return;
-      }
-      if (!validateFolderName(folderName)) {
-        return codebridgeI18n.invalidNameError();
-      }
-      const existingFolder = Object.values(projectFolders).some(
-        f => f.name === folderName && f.parentId === parentId
-      );
-      if (existingFolder) {
-        return codebridgeI18n.folderExistsError();
-      }
-    },
-  });
-  if (results.type !== 'confirm') {
-    return;
-  }
-  const folderName = extractDialogClosePromiseInput(results);
-
-  const folderId = getNextFolderId(Object.values(projectFolders));
-  newFolder({parentId, folderName, folderId});
-
-  const eventName =
-    parentId === DEFAULT_FOLDER_ID
-      ? EVENTS.CODEBRIDGE_NEW_FOLDER
-      : EVENTS.CODEBRIDGE_NEW_SUBFOLDER;
-  sendCodebridgeAnalyticsEvent(eventName, appName);
-};*/
-
-/*
-type OpenNewFilePromptArgsType = {
-  parentId: FolderId;
-  appName: string;
-  dialogControl: DialogControlInterface;
-  newFolder: NewFolderFunction;
-  projectFolders: ProjectType['folders'];
-};*/
-
-/*
-export interface DialogControlInterface {
-  closeDialog: DialogCloseFunctionType;
-  showDialog: (args: TypedDialogProps) => Promise<DialogClosePromiseReturnType>;
-  deferredPromiseObject: DeferredPromiseObject;
-  promiseArgs: unknown;
-  setPromiseArgs: (args: unknown) => void;
-}*/
-
-/*export const validateNewFolderName = ({
-  folderName,
-  parentId,
-  projectFolders,
-}: ValidateInputArgs) => {
-  if (!folderName.length) {
-    return;
-  }
-  if (!validateFolderName(folderName)) {
-    return codebridgeI18n.invalidNameError();
-  }
-
-  const existingFolder = Object.values(projectFolders).some(
-    f => f.name === folderName && f.parentId === parentId
-  );
-  if (existingFolder) {
-    return codebridgeI18n.folderExistsError();
-  }
-}; */
-
```

Comment:
This was all just commented code to help me structure it when writing the tests, I just forgot to delete it initially.